benchs/base.py

```diff
@@ -184,3 +205,15 @@ def is_dev_zoned(dev):
         res = f.readline()
 
     return ("host-managed" in res)
+
+# Checkout and build SPDK in Host System
+def spdk_build(spdk_build_script_path, install_dir, dev):
+    print("Using %s dir for SPDK (and FIO) builds" % install_dir)
```

Comment:
Could we use the f-string formatting instead of the %-formatting for new code changes? It is more readable in my opinion. :)
```suggestion
    print(f"Using {install_dir} dir for SPDK (and FIO) builds")
```

Reply:
Updated!